Project.toml

```diff
@@ -1,11 +1,12 @@
 name = "ReverseGeocode"
 uuid = "96b87e3f-b46e-456c-8178-0e438baab6fa"
 authors = ["Jan Mrkos <mrkosjan@gmail.com>"]
-version = "0.3.2"
+version = "0.3.3"
```

Comment:
Do you need to increase the patch version in your fork for your work? Anyway, since you are adding functionality, I think I will tag this v0.4.0 when it's merged into the main.

Reply:
So we have our in-house registry, I did that because I wasn't sure if you were still around to maintain the code, and wanted a fork that would track my changes.

Once this is merged, I'm more than happy to default back to this instead of the custom package